walkthroughs/week-1-web-development/examples/MyLife/src/main/webapp/about_me.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Nice job keeping the right indentation everywhere! :) Makes it easy for reviewers to read